charts/ingress-nginx/values.yaml

```diff
@@ -507,6 +509,8 @@ controller:
       enabled: false
       # -- Annotations are mandatory for the load balancer to come up. Varies with the cloud service. Values passed through helm tpl engine.
       annotations: {}
+      # -- Disable load balancer NodePort allocation
```

Comment:
Maybe improve the comment here just for something like "set to false to disable load balancer NodePort allocation"